pkg/cmd/roachtest/test_runner.go

```diff
@@ -339,6 +358,9 @@ func (r *testRunner) Run(
 	if len(r.status.fail) > 0 {
 		return errTestsFailed
 	}
+	// To ensure all prometheus metrics have been scraped, ensure shutdown takes
+	// at least one scrapeInterval.
+	time.Sleep(prometheusScrapeInterval - timeutil.Since(shutdownStart))
```

Comment:
figure out how to avoid
```
23:36:15 main.go:482: runTests destroying all clusters
I230104 23:36:20.043812 1192 util/stop/stopper.go:560  [-] 1  quiescing...
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2  slow quiesce. stack traces:
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +goroutine 1239 [running]:
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +github.com/cockroachdb/cockroach/pkg/util/log.getStacks(0x1)
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	github.com/cockroachdb/cockroach/pkg/util/log/get_stacks.go:25 +0x89
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +github.com/cockroachdb/cockroach/pkg/util/log.DumpStacks({0x74d5558, 0xc0000ba090}, {0x5e0f7cc, 0xc})
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	github.com/cockroachdb/cockroach/pkg/util/log/clog.go:407 +0x50
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Quiesce.func2()
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:563 +0x2d
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +created by time.goFunc
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	GOROOT/src/time/sleep.go:176 +0x32
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +goroutine 1 [sleep]:
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +time.Sleep(0x4c4b40)
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	GOROOT/src/runtime/time.go:195 +0x135
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Quiesce(0xc000c65a00, {0x74d5558?, 0xc0000ba090})
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:584 +0x1a5
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc000c65a00, {0x74d5558, 0xc0000ba090})
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:526 +0x11d
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +main.runTests(0x61c9458, {{0xc0006e4230, 0x1, 0x7}, 0x1, 0x12c, 0x0, 0x0, 0x0, 0x1f90, ...})
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	main/pkg/cmd/roachtest/main.go:489 +0xb6f
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +main.main.func5(0xc000edb680?, {0xc0006e4230, 0x1, 0x7})
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	main/pkg/cmd/roachtest/main.go:233 +0x2bf
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +github.com/spf13/cobra.(*Command).execute(0xc000edb680, {0xc0006e41c0, 0x7, 0x7})
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	github.com/spf13/cobra/external/com_github_spf13_cobra/command.go:856 +0x67c
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +github.com/spf13/cobra.(*Command).ExecuteC(0xc000eda500)
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	github.com/spf13/cobra/external/com_github_spf13_cobra/command.go:974 +0x3bd
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +github.com/spf13/cobra.(*Command).Execute(...)
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	github.com/spf13/cobra/external/com_github_spf13_cobra/command.go:902
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +main.main()
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +	main/pkg/cmd/roachtest/main.go:364 +0x1515
I230104 23:38:15.044277 1239 util/log/clog.go:409  [-] 2 +
```